src/Native/Unix/CMakeLists.txt

```diff
@@ -231,3 +231,7 @@ add_subdirectory(System.Native)
 add_subdirectory(System.Net.Http.Native)
 add_subdirectory(System.Net.Security.Native)
 add_subdirectory(System.Security.Cryptography.Native)
+
+if(OSX)
+add_subdirectory(System.Security.Cryptography.Native.Apple)
```

Comment:
I'm not current on nomenclature here.  Are we now naming things "Apple" instead of "OSX"?